src/Apps/CollectorProfile/collectorProfileRoutes.tsx

```diff
@@ -175,8 +175,16 @@ export const collectorProfileRoutes: AppRouteConfig[] = [
           const featureFlag = featureFlags["force-enable-artworks-list"]
           const isFeatureFlagEnabled = featureFlag?.flagEnabled ?? false
           const isArtsyEmployee = isArtsyEmail(context?.user?.email ?? "")
+          const isIntegrityUser =
+            context?.user?.email === "cypress+test@example.com"
```

Comment:
Just like the admin check, this check can go away when we release the feature to all users.